common/scripted_triggers/hfe_trigger.txt

```diff
@@ -323,21 +328,30 @@ hfe_trigger_is_agrarian = {
 }
 
 hfe_trigger_is_clever = {
-	or = {
+	OR = {
 		has_trait = trait_intelligent
 		has_trait = trait_natural_engineers
 		has_trait = trait_natural_physicists
 		has_trait = trait_natural_sociologists
 		has_trait = trait_presapient_natural_intellectuals
 		has_trait = trait_erudite
 		has_trait = hfe_species_trait_clever_beast
+		has_trait = trait_enigmatic_intelligence
+		has_trait = trait_enigmatic_intelligence_poor
+	}
+	NOT = {
+		has_trait = trait_nerve_stapled
+		has_trait = hfe_species_trait_happy_slave
+		has_trait = trait_presapient_docile_livestock
 	}
 }
 
 hfe_trigger_is_robot = {
 	or = {
 		has_trait = trait_mechanical
 		has_trait = trait_machine_unit
+		species = { is_archetype = ROBOT }
```

Comment:
redundant

Reply:
See above, on the Machine Archetype comment